.travis.yml

```diff
@@ -108,33 +109,11 @@ matrix:
     - if: type = cron OR commit_message =~ /\[full build\]/
       env: MODULE='cassandra' CASSANDRA_VERSION='3.0.14' ARGS='-Dtest=**/cassandra/*,**/utils/*'
     - if: type = cron OR commit_message =~ /\[full build\]/
-      env: MODULE='cql' CASSANDRA_VERSION='3.0.14' CASSANDRA_ENABLE_BOP='true' ARGS='-Dtest.skip.murmur=true -Dtest.skip.murmur-ssl=true -Dtest.skip.murmur-serial=true'
+      env: MODULE='cql' ARGS=' -Pcassandra3-byteordered -Dcassandra.docker.version=3.0.18'
     - if: type = cron OR commit_message =~ /\[full build\]/
-      env: MODULE='cql' CASSANDRA_VERSION='3.0.14' ARGS='-Dtest.skip.byteorderedpartitioner=true -Dtest.skip.murmur-ssl=true'
+      env: MODULE='cql' ARGS=' -Pcassandra3-murmur -Dcassandra.docker.version=3.0.18'
     - if: type = cron OR commit_message =~ /\[full build\]/
-      env: MODULE='cql' CASSANDRA_VERSION='3.0.14' CASSANDRA_ENABLE_SSL='true' ARGS='-Dtest.skip.murmur=true -Dtest.skip.byteorderedpartitioner=true -Dtest.skip.murmur-serial=true'
-    - if: type = cron OR commit_message =~ /\[full build\]/
-      env: MODULE='hadoop-parent/janusgraph-hadoop-2' CASSANDRA_VERSION='2.2.10' ARGS='-DskipHBase'
-    - if: type = cron OR commit_message =~ /\[full build\]/
-      env: MODULE='cassandra' CASSANDRA_VERSION='2.2.10' CASSANDRA_ENABLE_BOP='true' ARGS='-Dtest=**/thrift/* -Dtest.skip.unordered=true -Dtest.skip.ssl=true -Dtest.skip.serial=true'
-    - if: type = cron OR commit_message =~ /\[full build\]/
```

Comment:
Why you removed all cassandra 2 tests from `[full build]?`

Reply:
They are duplicated. The default version of Cassandra is in branch 2.2. So they are tested already with the default tests of cassandra and cql.